src/render.rs

```diff
@@ -139,10 +139,13 @@ impl<'a> Default for MarkdownRenderer<'a> {
 /// use render::markdown_to_html;
 ///
 /// let text = "[Rust](https://rust-lang.org/) is an awesome *systems programming* language!";
-/// let rendered = markdown_to_html(text)?;
+/// let rendered = markdown_to_html(text, None)?;
 /// ```
-pub fn markdown_to_html(text: &str) -> CargoResult<String> {
-    let renderer = MarkdownRenderer::new();
+pub fn markdown_to_html(text: &str, repo: Option<&str>) -> CargoResult<String> {
```

Comment:
Can you add a note in the documentation about the intended usage & the expected format of the new parameter?